src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -254,22 +254,33 @@ protected Task Stop()
                     [String]$CommandName
                 )
                 try {
-                    $null = Microsoft.PowerShell.Core\Get-Command $CommandName -ErrorAction Stop
+                    $command = Microsoft.PowerShell.Core\Get-Command $CommandName -ErrorAction Stop
                 } catch [System.Management.Automation.CommandNotFoundException] {
                     $PSCmdlet.ThrowTerminatingError($PSItem)
                 }
                 try {
-                    $null = Microsoft.PowerShell.Core\Get-Help $CommandName -Online
-                } catch [System.Management.Automation.PSInvalidOperationException] {
-                    Microsoft.PowerShell.Core\Get-Help $CommandName -Full
-                }";
+                    $helpUri = [Microsoft.PowerShell.Commands.GetHelpCodeMethods]::GetHelpUri($command)
+                    # HEAD means we don't need the content itself back :)
+                    $status = (Invoke-WebRequest -Method Head -Uri $helpUri -ErrorAction Stop).StatusCode
```

Comment:
What will happen when Windows PowerShell can't get the page because Tls12 isn't enabled? Should we be adding `[System.Net.ServicePointManager]::SecurityProtocol += "Tls12"` to be safe?

Are there implications of adding Tls12 here? I understand that it could break some older sites which might be a concern for some users.

Reply:
We can wrap it in a `try`/`finally` and set it back after the request is made. Hopefully that's not too expensive to accomplish.

Reply:
Since we're only requesting a site that MS owns, that should be guaranteed to work :)